cmd/utils/flaggroup.go

```diff
@@ -119,6 +119,7 @@ var FlagGroups = []FlagGroup{
 	{
 		Name: "DATABASE MIGRATION",
 		Flags: []cli.Flag{
+			CleanDBDirNameFlag,
```

Comment:
@winnie-byun How about removing this flag and setting this feature as a default?

Reply:
That is a good idea. I applied your suggestion. PTAL :)